recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 test:
     commands:
         - test -f ${PREFIX}/lib/libuuid.a  # [unix]
-        - conda inspect linkages -n _test libuuid  # [unix]
+        - conda inspect linkages libuuid  # [unix]
```

Comment:
We skip the recipe on Windows anyway.

```
skip: True  # [win]
```

what does inspect objects do? I've never used it, inspect linkages works on macOS just fine btw.